crates/db_schema/src/source/user_captcha.rs

```diff
@@ -0,0 +1,19 @@
+#[cfg(feature = "full")]
+use crate::schema::user_captcha;
+use serde::{Deserialize, Serialize};
+use serde_with::skip_serializing_none;
+
+#[skip_serializing_none]
+#[derive(Clone, PartialEq, Eq, Debug, Serialize, Deserialize)]
+#[cfg_attr(feature = "full", derive(Queryable, Insertable, AsChangeset))]
+#[cfg_attr(feature = "full", diesel(table_name = user_captcha))]
+pub struct UserCaptcha {
+  pub uuid: String,
+  pub answer: String,
+  pub expires: chrono::NaiveDateTime,
+}
+
+pub struct CheckCaptcha {
+  pub uuid: String,
+  pub answer: String,
+}
```

Comment:
This struct doesnt add anything useful. Removing it should also allow to get rid of a couple clones.